src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -3,21 +3,32 @@ layout: article
 title: Non-interactive
 description: Deploy changes to Appwrite projects to migrate databases and collections schema, functions, teams, buckets, and more.
 ---
-The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server. You can enable the non-interactive mode for the Appwrite CLI by setting the `project ID`, `endpoint`, and `API Key`:
+
+The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server.
+
+# Authentication {% #authentication %}
+
+In non-interactive mode, you can authenticate your users using two methods: email and password or API keys. While using an API key does not require an email and password to authenticate users, the API key needs to be generated and does not work for project API calls.
```

Comment:
> API keys

Should this be singular since you're only passing 1 API key?

> the API key needs to be generated

This is pretty obvious but let's mention scopes as that isn't as obvious.

> project API calls.

This can be confusing because list documents can be interpreted as a project API call. Let's clarify it by specifying API calls that update project settings.